model/AdminService.php

```diff
@@ -92,4 +93,13 @@ public static function addPermissionToClass(\core_kernel_classes_Class $class, $
             self::addPermissionToClass($subclass, $userUri, $rights);
         }
     }
+    
+    public static function getAdminAction() {
+        $classAdminXml = simplexml_load_string(
+            '<action id="access-control-admin" name="Access control" url="/taoDacSimple/AdminAccessController/adminPermissions" group="tree" context="resource">'
+            .'<icon id="icon-unlock" />'
+            .'</action>'
+        );
+        return Action::fromSimpleXMLElement($classAdminXml);
```

Comment:
May be at some point it would be better to create actions without having to use the XML format. 

In addition, is `name="Access control"` parsed by the translation script ?